internal/tensorzero-node/lib/bindings/OrderBy.ts

```diff
@@ -18,4 +18,5 @@ export type OrderBy = {
        */
       name: string;
     }
+  | { by: "term_frequency" }
```

Comment:
RFC: Just `search_relevance` so we can evolve under the hood however we'd like? Later maybe more granular options.

Reply:
Done.